lib/iris/tests/unit/analysis/test_VARIANCE.py

```diff
@@ -23,13 +23,13 @@
 # importing anything else.
 import iris.tests as tests
 
-from iris._lazy_data import as_lazy_data
 import numpy as np
 import numpy.ma as ma
 
 from iris.analysis import VARIANCE
 import iris.cube
 from iris.coords import DimCoord
+from iris._lazy_data import as_lazy_data, as_concrete_data
```

Comment:
We do need to agree on an order for importing hidden modules, atm we're inconsistent with it. 
Unfortunately, a quick google didn't unearth any standard practice
I do like this approach of putting them in alphabetical, regardless of whether or not they are private though